docs/ruleset_functions/G311_exceptions/does_zone_meet_G3_1_1e.md

```diff
@@ -0,0 +1,54 @@
+# does_zone_meet_G3_1_1e
+**Schema Version:** 0.0.22  
+
+**Description:** determines whether a given zone meets the G3_1_1e exception "Thermal zones designed with heating-only systems in the proposed design serving storage rooms, stairwells, vestibules, electrical/mechanical rooms, and restrooms not exhausting or transferring air from mechanically cooled thermal zones in the proposed design shall use system type 9 or 10 in the baseline building design."
+
+**Inputs:**
+- **P-RMI**
+- **B-RMI**
+- **zone_id**
+
+**Returns:**  
+- **result**: an ENUM YES, NO, or MAYBE_VESTIBULE
+ 
+**Function Call:**
+- **is_a_vestibule**
+- **is_zone_mechanically_heated_and_not_cooled**
+- **get_component_by_id**
+- **does_zone_get_cooled_transfer_air** - placeholder function
+
+## Logic:
+- create list of eligible space types: `eligible_space_types = ["STORAGE_ROOM_HOSPITAL", "STORAGE_ROOM_SMALL", "STORAGE_ROOM_LARGE", "WAREHOUSE_STORAGE_AREA_MEDIUM_TO_BULKY_PALLETIZED_ITEMS", "WAREHOUSE_STORAGE_AREA_SMALLER_HAND_CARRIED_ITEMS", "STAIRWELL", "ELECTRICAL_MECHANICAL_ROOM", "RESTROOM_FACILITY_FOR_THE_VISUALLY_IMPAIRED", "RESTROOM_ALL_OTHERS"]
+- set the result variable to No - only a positive test can give it a different value: `result = NO`
+- set eligibility boolean to TRUE: `eligible = TRUE`
+- get the zone in the P-RMI: `zone_p = get_component_by_id(P-RMI,zone_id)`
+- get the zone in the B-RMI: `zone_b = get_component_by_id(B-RMI,zone_id)`
+- For each space in zone_b: `for space_p in zone_b.Spaces:`
+	- check if the space has an eligible lighting type: `if space_p.lighting_space_type not in eligible_space_types:`
```

Comment:
@KarenWGard the logic tends to go back and forth in the maybe vestibule and transfer air section. would it make sense to first check for transfer air and then do the final check for maybe vestibule?

Reply:
@supriyagoel - the `is_zone_a_vestibule` function is only being called once.  I am assigning the result of this function to the variable `is_a_vestibule`

Regarding transfer air, the function `is_zone_mechanically_heated_and_not_cooled` checks for cooled transfer air, so we don't need to use the function `does_zone_get_cooled_transfer_air`.  I removed the last two lines.

the logic is now:
1. lighting space type check
2. heated and not cooled check (includes transfer air check)
3. vestibule check